src/components/controller/rightController/rightController.css

```
@@ -0,0 +1,64 @@
+.rightController {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  background-color: turquoise;
+  padding: 50px;
+  border-radius: 0 25px 25px 0;
+}
+
+.listWrap {
+  background-color: black;
+  width: 200px;
+  padding: 20px;
+  color: yellow;
+}
+
+.scoreList > li {
+  text-align: end;
+}
+
+.keys > button {
+  border-radius: 100%;
+  background: linear-gradient(
+    0deg,
+    rgba(46, 47, 69, 1) 0%,
+    rgba(94, 94, 150, 1) 75%
+  );
+  border: solid 3px rgb(46, 47, 69);
+  outline-style: none;
+}
+
+#btn-up-right {
+  grid-area: 1 / 3 / 2 / 4;
+  background: linear-gradient(
+    0deg,
+    rgba(46, 47, 69, 1) 0%,
+    rgba(94, 94, 150, 1) 75%
+  );
+}
+
+#btn-up-right:active {
+  background: linear-gradient(
+    180deg,
+    rgba(46, 47, 69, 1) 0%,
+    rgba(94, 94, 150, 1) 75%
+  );
+}
+
+#btn-down-left {
```

```suggestion
#btn-b {
```
hier würde ich das gleiche wie bei btn-up-right empfehlen